TESTING.md

```
@@ -0,0 +1,353 @@
+open coverage/index.html
+```
+
+---
+
+## Test File Structure
+
+```
+ehr-proxy/tests/
+├── conftest.py                    # Fixtures and --live flag
+├── pytest.ini                     # Pytest configuration
+│
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)